test/model/src/core/util.cpp

```diff
@@ -373,6 +373,27 @@ parse_uint64(const char *str, const char **end)
     return r;
 }
 
+/*
+ * quote --
+ *     Return a string in quotes, with appropriate escaping.
+ */
+std::string
+quote(const std::string &str)
+{
+    std::ostringstream out;
+    out << "\"";
+
+    for (char c : str) {
+        if (c == '\"')
```

Comment:
The \ is not needed.  See test program below.

character.cpp
#include <iostream> /* std::cout */
#include <ostream> /* std::endl */
int main(int argc, char **argv) {
   char character = '"';
   std::cout << "charcter=" << character << " = " << (unsigned) character << std::endl;}

$ g++ character.cpp
$ ./a.out
charcter=" = 34

Reply:
Good point! Fixed in [6ea43c7](https://github.com/wiredtiger/wiredtiger/pull/10943/commits/6ea43c73999e1936f25c20d81eb3f8004acd0a3a).